examples/janus-gateway/streaming/main.go

```diff
@@ -52,29 +89,7 @@ func main() {
 		fmt.Printf("Connection State has changed %s \n", connectionState.String())
 	})
 
-	peerConnection.OnTrack(func(track *webrtc.Track, receiver *webrtc.RTPReceiver) {
-		if track.Codec().Name == webrtc.Opus {
-			return
-		}
-
-		fmt.Println("Got VP8 track, saving to disk as output.ivf")
-		i, err := ivfwriter.New("output.ivf")
-		if err != nil {
-			panic(err)
-		}
-
-		for {
-			packet, err := track.ReadRTP()
-			if err != nil {
-				panic(err)
-			}
-
-			err = i.AddPacket(packet)
-			if err != nil {
-				panic(err)
-			}
-		}
-	})
+	peerConnection.OnTrack(handleTrack)
```

Comment:
Do you mind unfolding `handleTrack` it just makes it easier for newer programmers to read (I think)